plutus-ledger/plutus-ledger.cabal

```diff
@@ -66,22 +70,23 @@ library
         Data.Aeson.Extras,
         Data.Either.Extras,
         Data.Text.Prettyprint.Doc.Extras,
-        Plutus.V1.Ledger.Address as Ledger.Address,
         Plutus.V1.Ledger.Ada as Ledger.Ada,
         Plutus.V1.Ledger.Bytes as Ledger.Bytes,
-        Plutus.V1.Ledger.Contexts as Ledger.Contexts,
         Plutus.V1.Ledger.Credential as Ledger.Credential,
         Plutus.V1.Ledger.DCert as Ledger.DCert,
-        Plutus.V1.Ledger.Crypto as Ledger.Crypto,
         Plutus.V1.Ledger.Interval as Ledger.Interval,
-        -- We manually re-export that module so we can include some extra
-        -- instances
-        -- Plutus.V1.Ledger.Scripts as Ledger.Scripts,
         Plutus.V1.Ledger.Slot as Ledger.Slot,
-        Plutus.V1.Ledger.Tx as Ledger.Tx,
         Plutus.V1.Ledger.TxId as Ledger.TxId,
         Plutus.V1.Ledger.Time as Ledger.Time,
         Plutus.V1.Ledger.Value as Ledger.Value
+        -- We manually re-export these modules so we can include some functions
+        -- that don't need to used outside of plutus:
```

Comment:
This comment can move above l. 73

Reply:
Actually, this comment is about module exports like `Plutus.V1.Ledger.Address as Ledger.Address,`, not module exports like `Plutus.V1.Ledger.Value as Ledger.Value`. The reason is that if we have a module with the same name between `plutus-ledger` and `plutus-ledger-api`, then there will be module clashing because of the cabal file exports. Originally, the goal was that if you import `plutus-ledger`, you also import `plutus-ledger-api`. However, now, because I had to create modules with the same name in `plutus-ledger`, I had to remove those modules from the cabal file and do:

```
module Ledger.Address
  ( module Export
  ...
  )
  import Plutus.V1.Ledger.Address as Export
...
```

So the question is: should we leave this comment (or clarify it)? And should we leave those commented modules down bellow as a way of documenting which of the modules from `plutus-ledger-api` are exported from the Haskell code in `plutus-ledger`.

Hope my ramble makes sense :)

Reply:
yeah I saw the re-exported modules. Maybe you can just add a line saying that the other modules are exported from within the Haskell modules, not here in cabal.